tests/src/test/scala/runtime/actionContainers/NodeJsActionContainerTests.scala

```diff
@@ -585,4 +586,40 @@ abstract class NodeJsActionContainerTests extends BasicActionRunnerTests with Ws
         e shouldBe empty
     })
   }
+
+  it should "not override runtime packages with user provided packages" in {
+    withContainer(nodejsTestDockerImageName) { c =>
+      val code = """
+                   | function main(args) {
+                   |  var ow2 = require('openwhisk');
+                   |  var ow3 = require('../node_modules/openwhisk')
```

Comment:
This test has two concerns, I think the test should validate that openwhisk 2.x is possible.
And there are two flavors via Dockerfile, which this one covers, and a new test via zip.
And not test the existence of of v3, maybe in another test to test but not 100% sure.